connegp-implementation-report/index.html

```diff
@@ -126,13 +126,13 @@ <h3>Functional Profiles Identified</h3>
             </tr>
             <tr>
               <td><code><span style="white-space: nowrap"><a href="http://www.w3.org/ns/dx/conneg/profile/qsa">cnpr:qsa</a></span></code></td>
-              <td>QSA Functional Profile</td>
-              <td>For systems negotiating for content with the use of URI [[RFC3986]] Query String Arguments and using the <code>_profile</code> argument key and <code>_profile=all</code> key/value pair</td>
+              <td>URL QSA Functional Profile</td>
+              <td>For systems negotiating for content with the use of URL [[RFC3986]] Query String Arguments and using the <code>_profile</code> argument key and <code>_profile=all</code> key/value pair</td>
```

Comment:
The mention of RFC 3986 in the context of URLs is not really correct since that is the **Uniform Resource Identifier (URI): Generic Syntax** specification. Perhaps "the use of URL Query String Arguments (cf. [[RFC3986]] for a definition of Query String Arguments in the context of URIs)".

Reply:
Done